<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDevTest</class>
 <widget class="QWidget" name="FormDevTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>设备名称</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QToolButton" name="pushButton_fdTest">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>费显显示器</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="pushButton_cardreader">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>卡读写器</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QToolButton" name="pushButton_ioTest">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>IO板</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QToolButton" name="pushButton_WtTest">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>计重设备</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QToolButton" name="pushButton_VprTest">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>车牌识别</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QToolButton" name="pushButton_VedioCardTest">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>摄像头</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QToolButton" name="pushButton_PrinterTest">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>打印机</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>设备测试区</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="QWidget" name="page_3"/>
        <widget class="QWidget" name="page_4"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
